src/selectivity-dropdown.js

```diff
@@ -78,6 +79,9 @@ function SelectivityDropdown(options) {
 
     if (options.showSearchInput) {
         selectivity.initSearchInput(this.$('.selectivity-search-input'));
+
+        this.$('.selectivity-search-input').blur(this._blurCloseProxy);
```

Comment:
Is this compatible with Zepto.js? I think `this.$('.selectivity-search-input').on('blur', this._blurCloseProxy);` might be the way to go.